lib/validators/options_processor/options/lib/andOrProcessor.js

```diff
@@ -24,8 +24,15 @@ const andOrProcessor = (validatorContext, input, rule, key) => {
   let isMergeable
   let DEFAULT_SUCCESS_MESSAGE = `All the requisite validations passed for '${key}' option`
   let descriptions = ''
+  let errorExists = false
```

Comment:
Should this variable be `isErrorExists` ? This would be inline with our naming convention for boolean type variables as declared on line 27. `isMergeable`